lib/decorators/formatted_dates.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class FormattedDates < Scraped::Response::Decorator
```

Comment:
I think you need to add some documentation here to explain what this Decorator is doing.